data/upload_index_data.py

```diff
@@ -34,33 +34,42 @@ def prep_data(json_file_name: str):
     """
     Prepare dataset for upload to Azure index
     """
-    # Reformat data to {text, metadata} structure
-    docs = pd.read_json(f'{json_file_name}.json')
-    docs['metadata'] = docs['Level1'] + ";" + docs['art']
-    docs = docs.drop(columns=['Level1', 'art', 'p_num'])
-    documents = docs.to_dict('records')
+    f = open(f'{json_file_name}.json')
+    # returns JSON object as a dictionary
+    documents = json.load(f)
 
     # Prepare data for upload
     docs = []
```

Comment:
maybe consider changing the name of this variable, not to have the same name as the on in the global context. It works for now but it may create a problem in the future.